### ISSUE TITLE
week 15 games not visible?

### ISSUE THREAD

**Issue by suikerchiller** [aqw/xbmc-gamepass#374]
Anyone else having issues with live games at the moment? The game listing of week 15 is not visible at all.. The other weeks work fine though. Red zone / NFL network also work.

Running latest stable version of libreelec on raspberry pi3.

**Comment by TweekDash** [comment]
This happens to me often, it disappears but the later shows up fine. Redzone works when this is happening. 


I get an error if I select week 15 will post log later

**Comment by Displayel** [comment]
Is there wnyway of fixing the games crashing back to the week overview all the time, i think this happens about 10 times on sundays...not very good

**Comment by emilsvennesson** [comment]
Can you please try with the latest master and see if the problem still occurs?

**Comment by TweekDash** [comment]
I was fine for the Saturday games and the morning games. Then when the afternoon games started I got the error. I uninstalled my gamepass and installed the new master. I rebooted my nvidia shield, started kodi and gamepass and got the error again. I quit kodi, opened kodi turned on log debugging so I could provide a log to this thread and when I opened gamepass again the error was gone and the current week loaded fine.


Hope this helps a little.
